nengo_viz/static/viz_sim_control.js

```diff
@@ -182,23 +173,6 @@ VIZ.TimeSlider = function(args) {
     /** scale to convert time to x value (in pixels) */
     this.kept_scale = d3.scale.linear();
     
-    /** create the overall div */
-    this.div = document.createElement('div');
```

Comment:
Oh this might be a view thing again, this code was in the constructor for TimeSlider, not in the on_resize function